config/kubernetes/sig-docs/teams.yaml

```diff
@@ -1,4 +1,18 @@
 teams:
+  sig-docs-bn-owners:
+    description: Admins for Bengali content
```

Comment:
In line with the rest of the document, this should read as

```suggestion
    description: Approvers for Bengali content
```

Reply:
Okay sure!